subiquity/controllers/filesystem.py

```diff
@@ -139,52 +71,94 @@ def __init__(self, app):
         self.answers.setdefault('guided', False)
         self.answers.setdefault('guided-index', 0)
         self.answers.setdefault('manual', [])
-        self._cur_probe = None
         self._monitor = None
-        self._udev_listen_handle = None
-        self._probes = {}
+        self._crash_reports = {}
+        self._probe_once_task = SingleInstanceTask(self._probe_once)
+        self._probe_task = SingleInstanceTask(self._probe)
+
+    async def _probe_once(self, restricted):
+        if restricted:
+            probe_types = {'blockdev'}
+            fname = 'probe-data-restricted.json'
+            key = "ProbeDataRestricted"
+        else:
+            probe_types = None
+            fname = 'probe-data.json'
+            key = "ProbeData"
+        debug_flags = self.debug_flags
+        if 'bpfail-full' in debug_flags and not restricted:
+            await asyncio.sleep(2)
+            1/0
```

Comment:
note to self, this is intentional crash, due to crash in debug flags.

Reply:
I'd actually like to push this crash further down, into the prober but well. One thing at a time.